lib/Settings/Setting_wrapper.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:cboard_mobile/Settings/ThemeManager.dart';
+import 'package:cboard_mobile/Settings/routes.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'routes.dart';
+
+class Setting_wrapper extends StatelessWidget {
+  // ThemeNotifier theme = new ThemeNotifier();
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(
+        primaryColor: Color(0xFF23036A),
+        accentColor: Color(0xFFE3CAFF),
+        visualDensity: VisualDensity.adaptivePlatformDensity,
+        scaffoldBackgroundColor: Colors.white,
+      ),
+      home: Scaffold(
+        backgroundColor: Color(0xffE5E5E5),
+        appBar: AppBar(
+          centerTitle: true,
+          title: Text('Settings'),
+        ),
+        body: MyStatefulWidget(),
+      ),
+      routes: getRoute(),
+    );
+  }
+}
+
+class MyStatefulWidget extends StatefulWidget {
+  MyStatefulWidget({Key key}) : super(key: key);
+  @override
+  _MyStatefulWidget createState() => _MyStatefulWidget();
+}
+
+class _MyStatefulWidget extends State<MyStatefulWidget> {
+  final sections = ['People', 'Language', 'System', 'Help'];
+  final List<Map<String, List<Object>>> features = <Map<String, List<Object>>>[
+    {
+      'Guest': [Icons.account_circle]
+    },
+    {
+      'Language': [Icons.language, '/speech'],
+      'Speech': [Icons.record_voice_over, '/speech']
+    },
+    {
+      'Export': [Icons.cloud_upload, '/speech'],
+      'Import': [Icons.cloud_download, '/speech'],
+      'Display': [Icons.remove_red_eye, '/display'],
+      'Scanning': [Icons.center_focus_strong, '/speech'],
+      'Navigation': [Icons.chevron_right, '/navigation']
+    },
+    {
+      'User Help': [Icons.help, '/user help'],
+      'About Cboard': [Icons.info, '/about'],
+      'Donate': [Icons.monetization_on, '/speech'],
+      'Feedback': [Icons.feedback, '/speech']
+    },
+  ];
+  @override
+  Widget build(BuildContext context) {
+    return ListView.separated(
+      separatorBuilder: (BuildContext context, int index) {
+        return SizedBox(
+          height: 10,
+        );
+      },
```

Comment:
I see you used a `SizedBox` as a separator here - I used a bottom margin instead. Which one do you think we should use instead for consistency?

Reply:
I'm not sure about this. If I use bottom margin, then the `Help` section will have blank space at the bottom, which looks kinda weird.

Reply:
Ohh, yeahh - that's right! Thanks for pointing that out - in that case, `SizedBox` makes more sense then!